apps/src/lab2/views/dialogs/ShareDialog.module.scss

```diff
@@ -61,19 +51,47 @@
       gap: 20px;
     }
 
-    .careers {
-      background-color: $neutral_dark80;
+    .block {
+      background-color: $neutral_dark90;
       padding: 20px;
       border-radius: 10px;
       color: white;
       display: flex;
       flex-direction: column;
       gap: 20px;
+      height: 100%;
+
+      &Heading {
+        text-align: center;
+      }
+
+      &Afe {
+        background-color: #2E4486;
```

Comment:
Out of curiosity is this a color provided by AFE? Or can we use one of our constants?

Reply:
It came from an internal design comp, and I didn't see a match.  Let's keep an eye on this as we make revisions.